data/presets/presets/emergency/water_tank.json

```diff
@@ -0,0 +1,20 @@
+{
+    "icon": "water",
+    "fields": [
+        "name",
+        "operator",
+        "ref"
+    ],
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "terms": [
+        "cistern",
+        "reservoir"
+    ],
+    "tags": {
+        "emergency": "water_tank"
+    },
+    "name": "Water Tank"
```

Comment:
Can you change this to "Emergency Water Tank" ? 

I wouldn't want people to confuse it with just a normal water tank like: https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dstorage_tank

Reply:
Good point, I have changed the name accordingly and downgraded "water tank" to a search term.